components/inputs/input-date.js

```diff
@@ -313,6 +313,14 @@ class InputDate extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(LitEl
 		if (this._textInput) this._textInput.focus();
 	}
 
+	async open() {
+		if (!this._dropdownFirstOpened) await this._handleFirstDropdownOpen();
+		this._dropdown.open();
+		await this._handleChange();
+		this._calendar.focus();
+		this._setFormattedValue();
+	}
```

Comment:
What do you think about making this an `opened` attribute/property similar to how dialogs work? Then internally it could be refactored a bit to just use that.

IMO we should always be relying on properties to store state like this... it just makes the component so much easier to use vs. having to get a reference to it in JS and call a method on it.

Reply:
Yup that makes sense to me